modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,91 @@
+* Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus.
+* New `font_size_input_default_unit` option allows entry of numbers without a unit in `fontsizeinput`. They are then parsed as the set unit. If `font_size_input_default_unit` is not set the default is `pt`.
+* New `group` and `togglebutton` in view.
+* New `togglebutton` in dialog footer buttons.
+* Added `toggleFullscreen` to dialog API.
+* New `text-size-increase` and `text-size-decrease` icons.
+* New `xss_sanitization` option to allow disabling of XSS sanitization.
+* Added the top right close button of modal dialogs to the tabbing order. The 'x' button in these dialogs can now be accessed using keyboard navigation.
+* New `ui_mode` option for editor in scrollable containers support.
+
+=== Improved
+* Direct invalid child text nodes of list elements are now wrapped in list item elements.
+* Templates are now be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS.
+* Pressing backspace on an empty line now preserves formatting on the previous empty line.
+* Pressing enter inside the `inputfontsize` input field now moves focus back into the editor content.
+* Drag and drop events for elements with a `contenteditable="false"` attribute now includes target element details.
+* Updated focus, active, and enabled colors of UI buttons for improved contrast against the UI color.
+
+=== Changed
+* The `link` plugins context menu items no longer appears for links that include elements with a `contenteditable="false"` attribute.
+* The formatting of elements with a `contenteditable="false"` attribute are no longer cloned to new cells when new table rows are created.
+* Changed the color of `@dialog-table-border-color`, and added right padding to the first cell of dialog table.
+* The sidebar element now has the accessibility role `region` when visible and the accessibility role `presentation` when hidden.
+* The `tox-custom-editor` class now has a border highlight when it is selected.
+
+=== Fixed
+* Add support for navigating inside the tree component using arrow keys and shift key.
+* Sometimes the editor would finish initializing before the silver theme would have finished loading.
+* The `searchreplace` modal closed incorrectly when clicking outside of the alert that pops up when no match is found.
+* The text color or background color picker toolbar buttons did not update when the text color or background color was changed using the equivalent commands in the Format menu.
+* The `onSetup` api function would not run when defining custom group toolbar button.
+* An element could be dropped onto the decendants of an element with a `contenteditable="false"` attribute.
+* Checkmark did not show in menu color swatches.
+* The foreground and background menu icons would not properly update to display the last used color.
+* Added new `setIconFill` function to `NestedMenuItemInstanceApi`.
```

Comment:
Move to `Added` section

Reply:
@tiny-ben-tran,

Is it safe to assume you want the equivalent change made to TinyMCE’s `changelog.md`?

Reply:
Moved.